pycdp/browser.py

```diff
@@ -110,6 +111,13 @@ def kill(self, timeout: float=3.0):
             if not self._keep_profile:
                 shutil.rmtree(self._profile, ignore_errors=True)
 
+    def __del__(self):
+        warnings.warn("The chrome instance was not closed properly with chrome.kill(); issuing SIGKILL.")

```

Comment:
we need to check if `self._process` is `None` because that means the process was already killed. `__del__` is always called when the chrome object is garbage collected, even if it was properly killed before.